pallets/template/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 
 /// Edit this file to define custom logic or remove it if it is not needed.
 /// Learn more about FRAME and the core library of Substrate FRAME pallets:
-/// <https://docs.substrate.io/reference/frame-pallets/>
+/// <https://docs.substrate.io/v3/runtime/frame>
```

Comment:
This is a regression. I think this is fixed upstream now in future releases
```suggestion
/// <https://docs.substrate.io/reference/frame-pallets/>
```